sec-keys/openpgp-keys-libxcrypt/openpgp-keys-libxcrypt-4.4.30.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="OpenPGP keys used by libxcrypt"
+HOMEPAGE="https://github.com/besser82/libxcrypt"
+SRC_URI="https://github.com/besser82/libxcrypt/releases/download/v${PV}/libxcrypt-gpgkey.gpg -> ${P}-libxcrypt-gpgkey.gpg"
```

Comment:
`libxcrypt` and `gpgkey` seem a bit redundant in the filename.

Also, this line is too long.

Reply:
sure, let's strip it.